### DISCUSSION TITLE
Turning your al-folio into a dual-language website | George C. de Araújo

### DISCUSSION THREAD

**Discussion by giscus** [george-gca/george-gca.github.io#15] [Announcements]
# Turning your al-folio into a dual-language website | George C. de Araújo

Adding support for another language in your al-folio.

https://george-gca.github.io/blog/2022/dual-language-al-folio/

**Comment by simonguichandut** [comment]
This is great, thank you! I was having some difficulties applying the translate function in all the right places, so I just ended up using [your repo as a template](https://github.com/george-gca/multi-language-al-folio). :)

**Comment by LiuYing-1** [comment]
Wuwuwu, sorry mate. I've tried and conducted the instructions given by you one by one. It failed and the error message is like "Permission Denied @sysopen". However, my project can be running in the beginning. I feel so frustrated, but I still show respect to your **clear and concise explanations**!

**Reply by george-gca** [reply]
Have you tried starting from my template repo? Since this is of interest to a lot of people, I decided to create a [multilingual version of al-folio](https://github.com/george-gca/multi-language-al-folio). I will update this post with the link to the template.

**Reply by george-gca** [reply]
Also, I recommend you do small incremental steps and test them individually. This way it is easier to check when an error occurs and why.

**Reply by LiuYing-1** [reply]
Morning! I haven't tried your template yet, and I will do it right now.
To your second reply, once I did a small change, I tested as well. I can't successfully get the result of "Header with Toggle" because of the failure. It returned the error like "Errno:EACCES" just after the sentence Loading translation .....
